balancer-js/src/modules/sdk.module.ts

```diff
@@ -44,6 +45,8 @@ export class BalancerSDK implements BalancerSDKRoot {
     public sor = new Sor(config),
     public subgraph = new Subgraph(config)
   ) {
+    const logger = Logger.getInstance();
+    logger.setLoggingEnabled(!!config.enableLogging);
```

Comment:
What does the `!!` operator do here?

Reply:
Shorthand to convert to a boolean:

In TypeScript, you can use the logical OR (||) operator along with the double exclamation mark (!!) to convert a boolean | undefined value to a boolean value in a shorthand way. This technique is called "double negation" or "double bang" operator. Here's an example:
```
typescript

const value: boolean | undefined = /* Some boolean or undefined value */;
const booleanValue: boolean = !!value;

console.log(booleanValue); // Outputs the boolean value (true or false)
```
In the above example, !!value is used to convert the boolean | undefined value value into a boolean value. If value is undefined, the first negation (!value) will result in true, and the second negation (!(!value)) will convert it back to false. If value is a boolean value, the double negation will preserve its original boolean value.

By using !!value, you can convert a boolean | undefined value to a boolean in a concise shorthand way.
(Thanks chatGPT :))

Reply:
Ok, [this post](https://dev.to/sanchithasr/what-is-the-double-bang-operator-in-javascript-4i3h) helped me understand.
Just out of curiosity, why use this `truthy` check instead of a `true` check? 🤔

Reply:
Thanks ChatGPT [2] 😂 
Ok, so in short, you're capturing `undefined` and considering it `false`.
Nice 😃